packages/terra-cell-grid/src/CellGrid.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import styles from './CellGrid.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * The child Cells for be passed to the cell grid.
```

Comment:
This description might need a little restructuring. Maybe `for` should be `to`?

Six year old me: `for be` ->

![image](https://user-images.githubusercontent.com/6720991/71480844-e062d900-27c0-11ea-8f59-888ca600b122.png)

Reply:
Updated.